### PR TITLE
Add Match Command, Bump to 0.1.13

### PR DESCRIPTION
**Pull request by hayesgm**
This patch adds a "match" command, that will verify if on-chain code matches your local build. This can be used as an alternative to code verification on Etherscan to make sure that the code is deployed matching a given version of this repository.

### CODE REVIEW

**Review by jflatow — APPROVED** [review]
OK so this version checks if your local code matches that at a particular address 👍 🆒

File: src/cli/commands/match.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    throw new Error(`Match Failed: Contract "${contractName}" creation reverted on ${network}`);
+  }
+
+  if (expectedBytecode != contractCode) {
+    error(diffStringsUnified(expectedBytecode, contractCode), verbose);
+
+    throw new Error(`Match Failed: Mismatched bytecode`);
+  }
+
+  info(`✅ Successfully matched ${contractName} to ${address} with args ${JSON.stringify(contractArgs)}`, verbose);
```
Comment (coburncoburn) [inline src/cli/commands/match.ts:43]: wondering if we will run into same issues as etherscan verifying, at least we can debug though!

**Review by coburncoburn — APPROVED** [review]

File: src/cli/commands/match.ts (hunk 1 above)
Comment (hayesgm) [reply src/cli/commands/match.ts:43]: It's been *super finicky* in my testing so far, honestly. If the code matches directly, it passes every time, but if I add another random contract that I don't even reference, it fails. It's almost like there's a checksum of the AST or something there. That said, I use jest-diff which pretty-prints the exact mismatch, so it makes it pretty easy to debug and spot and we can do it locally.

Comment (jflatow) [reply src/cli/commands/match.ts:43]: Feeling some sympathy for etherscan now?

**Pull request merged**